drydock/patches/kustomization

```diff
@@ -46,3 +46,60 @@ patches:
       value:
         argocd.argoproj.io/sync-wave: "5"
 {%- endif %}
+{% if DRYDOCK_ENABLE_CELERY_TUNING %}
+- path: plugins/drydock/k8s/celery-lms.yml
+- path: plugins/drydock/k8s/celery-cms.yml
+- path: plugins/drydock/k8s/forum-overrides.yml
+{% endif -%}
+{% if DRYDOCK_GRACEFUL_UWSGI -%}
+- target:
+    kind: Deployment
+    name: lms|cms
+  patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/lifecycle
```

Comment:
The same comment applies to these patches, we can consolidate them in a single file.